e2e/src/usecases/HandleDeepLinkSend.js

```diff
@@ -99,6 +100,13 @@ export default HandleDeepLinkSend = () => {
       await openDeepLink(PAY_URL)
       await expect(element(by.id('SendAmount'))).not.toBeVisible()
     })
+
+    it('should navigate to another screen correctly', async () => {
+      await openDeepLink(quote(deepLinks.navigateToSend))
+      await waitFor(element(by.id('RecipientPicker')))
+        .toBeVisible()
+        .withTimeout(10 * 1000)
+    })
```

Comment:
Great adding a test!
We should use a different test link though as we don't support opening arbitrary screens from deep links for security reasons.

https://github.com/valora-inc/wallet/blob/f3b008bb6297ec679419ad8604449e474029632a/src/app/saga.test.ts#L119-L126